src/components/Sticker/index.tsx

```diff
@@ -68,7 +89,30 @@ export default function Sticker({
             : null}
         </span>
       )}
-      {children}
+      {showTooltip ? (
+        <Popover
```

Comment:
might want to consider moving this to wrap the root div returned by this component. Currently if you hover over the edge of the sticker, it still shows the on hover effect but it does not show the tooltip.

The tooltip is only showing if you hover the text